src/features/game/events/buyWarBonds.test.ts

```diff
@@ -164,7 +164,34 @@ describe("buyWarBonds", () => {
       },
       action: { type: "warBonds.bought" },
     });
-
     expect(state.inventory["Human War Point"]).toEqual(new Decimal(12));
   });
+
+  it("mints extra items if they have an ancient warhammer", () => {
```

Comment:
Maybe just make the test description a bit more specific. `mints 10% more war bond tokens/points if they have an ancient warhammer`.